lib/storage.js

```diff
@@ -257,18 +265,16 @@ Storage.prototype.open = function (opts, cb) {
     })
   })
 
-  this.signatures.write(0, header(1, 64, 'Ed25519'), done)
+  this.signatures.write(0, header(1, this.signatureSize, 'Ed25519'), done)
```

Comment:
Does the const string here also need changing? Ie. are the ristretto255 points "decompressed" back into valid ed25519 points?

Reply:
Ah yes we should change the string too. Does the protocol use this to ensure peers are using the same kind of signatures?

Reply:
Not yet, but we’re working on updating the handshake to communicate info like this

Reply:
Added signatureType